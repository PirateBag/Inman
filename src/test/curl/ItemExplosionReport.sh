#/bin/bash
 curl -s -d '{ "parentId" : "1" }' -H "Content-Type: application/json" -X POST http://localhost:8080/itemReport/explosion
