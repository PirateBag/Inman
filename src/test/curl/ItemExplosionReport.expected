{
  "responseType" : "QUERY",
  "errors" : [ ],
  "data" : [ {
    "id" : 0,
    "activityState" : "NONE",
    "message" : "Summary     Description                      Unit Cost  Sourcing       Depthd"
  }, {
    "id" : 0,
    "activityState" : "NONE",
    "message" : "W-001       36 in red wagon                       0.00  MAN           0"
  }, {
    "id" : 0,
    "activityState" : "NONE",
    "message" : "   W-002       red painted wagon body                0.00  MAN           0"
  }, {
    "id" : 0,
    "activityState" : "NONE",
    "message" : "   W-003       Front Wheel assembly                  0.00  MAN           0"
  }, {
    "id" : 0,
    "activityState" : "NONE",
    "message" : "       W-004       24 in handle                          0.40  PUR           0"
  }, {
    "id" : 0,
    "activityState" : "NONE",
    "message" : "       W-005       8 in wheel                            0.50  PUR           0"
  }, {
    "id" : 0,
    "activityState" : "NONE",
    "message" : "   W-006       Rear Wheel assembly                   0.00  MAN           0"
  }, {
    "id" : 0,
    "activityState" : "NONE",
    "message" : "       W-007       Rear wheel bracket                    0.70  MAN           0"
  } ]
}